docs/pipelines/ecosystems/github-actions.md

```diff
@@ -85,21 +66,23 @@ If you have both Azure Pipelines and GitHub Actions workflows, you may want to t
                 azure-devops-token: ${{ secrets.AZURE_DEVOPS_TOKEN }}
     ```
 
-1. On the **Actions** page, verify that your workflow ran. Select the workflow title to see more information about the run. You should see a green check mark for the Azure Pipelines Action. Open the Action to see a direct link to the pipeline run. 
+1. Commit and push your workflow file.
+
+1. The workflow will run every time a commit is pushed to main or a pull request is opened against main. To verify that your action ran, open your GitHub repository and select **Actions**.
+
+    :::image type="content" source="media/github-actions-header.png" alt-text="Select Actions in the GitHub navigation"::: 
+
+1. Select the workflow title to see more information about the run. You should see a green check mark for the Azure Pipelines Action. Open the Action to see a direct link to the pipeline run. 
```

Comment:
This last step is a bit out of date, but I am not sure how to add new screenshots. Here are more up-to-date screenshots/text as an example: https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/viewing-workflow-run-history. (This change just pushed last week, so it is pretty new.)